### PR TITLE
Alphabetize help commands

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: src/oss/confluent.sh [hunk 1]
```diff
@@ -1406,42 +1406,42 @@ case "${command}" in
...
+        log_command "$@";;           
+
     start)
         start_command "$@";;
 
-    stop)
-        stop_command "$@";;
-
     status)
-        status_command "$@";;
```
Comment (ewencp) [inline src/oss/confluent.sh]: nit: seems some whitespace was accidentally added here

**Comment by joel-hamill** [comment]
@ewencp i vote for alphabetized (add DC/OS CLI to the list of alphabetized CLIs). i think if we go "most important" then it's bit subjective. one man or woman's "most important" command is another's least important. 

my $.02, in most cases people go to the help when they're stuck (then don't read from top-to-bottom) - so from the standpoint of a reference doc, we're making them do more work if our order has been determined by a certain groups view of "most important".

**Comment by ewencp** [comment]
@joel-hamill I'm fine either way, just wasn't sure if this was something @kkonstantine had thought about when listing the commands or if it just happened the way it did by chance of the ordering of additions to the tool

**Comment by joel-hamill** [comment]
@kkonstantine let me know what you think https://github.com/confluentinc/confluent-cli/pull/44/commits/e2fa8e983f7de80e7a66e8d185b1759c0822cf0d

**Comment by kkonstantine** [comment]
That looks kind of neat @joel-hamill 
I'll rebase it and remove some whitespace here and there, before I merge it. 

We can revisit classification into categories as the command set becomes richer. Thanks!

**Pull request merged**